### ISSUE TITLE
Hide Toolbar and Others(newbie user)

### ISSUE THREAD

**Issue by enemali** [audreyt/ethercalc#567]
Is there a way to 

1. Hide toolbar , formatting bar , formula bar and yet allow data entry
2.  Create a hyperlink to another cell within the same workbook , worksheet or another workbook
3.  Disable Right Click
4. How do i implement the Key Authentication discussed in Issue #1  ( i need a novice explanation please like go to this folder , open the file xxxx change or add this code to line xxxx , then run etthercalc at 
http://localhost:8000/  , then .................. 

pardon my elementary knowledge of ethercalc 

I am currently using ethercalc on my local-Pc with nodejs and redis server

**Comment by eddyparkinson** [comment]
>Hide toolbar , formatting bar , formula bar and yet allow data entry

I don't think this exists. If you can code, a small change to this would work https://ethercalc.org/ma7rils23qhq/view

>Create a hyperlink to another cell within the same workbook , worksheet or another workbook

"to another cell" is not possible but to another worksheet/workbook would be. just paste the link for tha sheet.

>Disable Right Click

Not possible without changing the code. But not hard to implement.

>How do i implement the Key Authentication discussed in Issue #1 ( i need a novice explanation please like go to this folder , open the file xxxx change or add this code to line xxxx , then run etthercalc at
http://localhost:8000/ , then ..................

Sanstorm version can do this. 
https://apps.sandstorm.io/app/a0n6hwm32zjsrzes8gnjg734dh6jwt7x83xdgytspe761pe2asw0

But sorry the best docs are:
https://github.com/audreyt/ethercalc/issues/1
https://github.com/audreyt/ethercalc/issues/4
 - if you post some better docs I can add them to the readme

**Comment by enemali** [comment]
Thanks for the feedback , i will take my time to read and understand #1  and #4  even though i noticed a lot of people find difficulties in implementing them.
I Just started using ethercalc so its likely i will ask a lot  of questions  (issues) here or is there a forum where ethercalc Q&A are being discussed?

**Comment by eddyparkinson** [comment]
yes - here is the best place to post.

**Comment by Tusharsb** [comment]
A quick hack to hide the menu and toolbar could be to insert the following lines of code in `static\makeup.css` file
```
/* Hides menu and toolbar */
#tableeditor > div:nth-child(1) > div:nth-child(1) {
    display:none;
}
```
A better solution would however be to 'introduce a new `command`' which when interpreted by ethercalc will toggle a 'fullscreen mode'.
